docs/CPP_STYLE.md

```diff
@@ -4,15 +4,15 @@ In general, [Google's coding standard](https://google.github.io/styleguide/cppgu
 
 Below are our specific (but not all!) exceptions to the Google's coding standard:
 
-- All C++ code should conform to the C++17 standard.
+- All C++ code should conform to the C++20 standard.
 - We use `.cpp` and `.hpp` files, not `.cc` and `.h` (`.c` and `.h` are used for C code), in UTF-8 encoding.
 - File names are lowercase with underscores, like `file_reader.cpp`.
 - We use `#pragma once` instead of the `#define` Guard in header files.
 - Includes are sorted and grouped by directory, there should be newlines between different directories.
 - Order of directories in includes: "current_dir/current_file.hpp", includes from other dirs sorted by dependencies (e.g. indexer, then coding, then base), "defines.hpp", C++ standard library headers, boost headers, 3party.
 - We ARE using C++ exceptions.
-- We are using all features of C++17 except the filesystem which is not fully supported on all platforms.
```

Comment:
Unfortunately, the filesystem remark is still relevant.
https://github.com/organicmaps/organicmaps/pull/8256#issuecomment-2134054267

Reply:
Damnit apple, will update

Reply:
Elementary string conversions std::to_chars, std::from_chars is also not available.

Reply:
noted, thanks